python/lsst/sims/utils/AstrometryUtils.py

```diff
@@ -1049,6 +1099,9 @@ def _icrsFromObserved(ra, dec, obs_metadata=None, epoch=None, includeRefraction=
     RA and the second row is the mean ICRS Dec (both in radians)
     """
 
+    are_arrays = _validate_inputs([ra, dec], ['ra', 'dec'], "icrsFromObserved")
+
```

Comment:
Again: just checking that inputs are self-consistent and raising an error if they are not